src/enclave/http.h

```diff
@@ -16,7 +16,7 @@ namespace enclave
       auto req = fmt::format(
         "POST / HTTP/1.1\r\n"
         "Content-Type: application/json\r\n"
-        "Content-Length: {}\r\n\r\n{}",
+        "Content-Length: {}\r\n\r\n",
```

Comment:
Why are we no longer including the POST body here?

Reply:
This was never used (see missing body arg to `fmt::format()`. Instead, the body of the command is sent in-line after the header in `send()`.

Reply:
No, in this instance (`post`) we _do_ pass a `body` arg, as well as the `body.size()` (cf `post_header` below, which I agree should remove the second `{}` since its never filled). It looks like this function is never actually called, so we should remove it to avoid this confusion.

Reply:
D'oh. You're absolutely right! As you suggested, I removed `post()` as it's not used.